### DISCUSSION TITLE
Why can't I edit notes in my Dataview project?

### DISCUSSION THREAD

**Discussion by marcusolsson** [marcusolsson/obsidian-projects#378] [FAQ]
Projects uses the Dataview API to retrieve results using a Dataview query. Unfortunately, as far as I know, the Dataview API returns little information about the source data, which means the Projects plugin can't know how to update a value given the results of a query.

## Problem 1

For example, let's say that you defined a project using the following Dataview query:

```
TABLE Status AS "Due", Due AS "Status" FROM "Notes"
```

In this case, the Projects plugin would update the wrong field since it would only know the aliased name. I've opened https://github.com/blacksmithgu/obsidian-dataview/issues/1572 in the Dataview repository for this to be added to Dataview.

## Problem 2

Projects also doesn't know whether a value came from an inline attribute or from a front matter property.

For example, given the query below, should Projects write the value as front matter or inline attributes?

```
TABLE Status FROM "Notes"
```

## Help wanted

If you're familiar with the Dataview API, and you want to help, let me know!

**Comment by hollowhemlock** [comment]
Have you checked out the plugin [metadatamenu](https://github.com/mdelobelle/metadatamenu)? I don't have experience with the api but maybe the plugin has an example of what your looking for. I think it has superseded [metaedit](https://github.com/chhoumann/MetaEdit).

**Comment by gig168** [comment]
Hello, I only learned about the Project plugin today and I must say it's amazing. My native language is not English, and I am using a translation software to communicate, please forgive any inconvenience.

Regarding this issue, I would like to offer an idea, which is to use Dataview solely to lock down the data source for new projects (to define exactly which .md files to process). For the operation of updating data, could it be possible to do so based on "folders" and "tags"?
